src/functions/bathymetry.ts

```diff
@@ -14,33 +16,36 @@ import project from "../../project";
 
 // @ts-ignore
 import geoblaze, { Georaster } from "geoblaze";
-import { clipSketchToGeography } from "../util/clipSketchToGeography";
-import { BathymetryResults, ExtraParams } from "../types";
-import { getParamStringArray } from "../util/extraParams";
+import { clipToGeography } from "../util/clipToGeography";
+import { BathymetryResults, DefaultExtraParams } from "../types";
 
 export async function bathymetry(
-  sketch: Sketch<Polygon> | SketchCollection<Polygon>,
-  extraParams?: ExtraParams
+  sketch:
+    | Sketch<Polygon | MultiPolygon>
+    | SketchCollection<Polygon | MultiPolygon>,
+  extraParams: DefaultExtraParams = {}
 ): Promise<BathymetryResults> {
-  const geographyId = extraParams
-    ? getParamStringArray("geographyIds", extraParams)[0]
-    : undefined;
-  const clippedSketch = await clipSketchToGeography(sketch, geographyId);
+  const geographyId = getFirstFromParam("geographyIds", extraParams);
+  const curGeography = project.getGeographyById(geographyId, {
+    fallbackGroup: "default-boundary",
+  });
+
+  const clippedSketch = await clipToGeography(sketch, curGeography);
```

Comment:
This function I left in project code, because it uses the projectClient.  It's very similar to a preprocessing function.  In fact a `clipToGeography` preprocessor could easily be created that wraps this function.  Anyway the main takeway is that clipToGeography will now be installed into all new projects on `init`.